pyvista/plotting/_property.py

```diff
@@ -398,6 +415,47 @@ def opacity(self, value: float):  # numpydoc ignore=GL08
         _check_range(value, (0, 1), 'opacity')
         self.SetOpacity(value)
 
+    @property
+    def edge_opacity(self) -> float:  # numpydoc ignore=RT01
+        """Return or set the edge opacity of this property.
+
+        Edge opacity of the mesh. A single float value that will be applied globally
+        edge opacity of the mesh and uniformly applied everywhere. Between 0 and 1.
+
+
+        Examples
+        --------
+        Set edge opacity to ``0.5``.
+
+        >>> import pyvista as pv
+        >>> prop = pv.Property()
+        >>> prop.show_edges = True
+        >>> prop.edge_opacity = 0.5
+        >>> prop.edge_opacity
+        0.5
+
+        Visualize default edge opacity of ``1.0``.
+
+        >>> prop.edge_opacity = 1.0
+        >>> prop.plot()
+
+        Visualize edge opacity of ``0.1``.
+
+        >>> prop.edge_opacity = 0.1
+        >>> prop.plot()
+
+        """
+        if vtk_version_info < (9, 3):
+            return 1.0
+        else:
+            return self.GetEdgeOpacity()
+
+    @edge_opacity.setter
+    def edge_opacity(self, value: float):  # numpydoc ignore=GL08
+        _check_range(value, (0, 1), 'edge_opacity')
+        if not vtk_version_info < (9, 3):
```

Comment:
more readable this way IMO
```suggestion
        if vtk_version_info >= (9, 3):
```